action_launcher.bash

```diff
@@ -27,7 +27,7 @@ if [ ! -f "$INPUT_CLANG_TIDY_FIXES" ]; then
   exit 0
 fi
 
-/action/run_action.py \
+${GITHUB_ACTION_PATH}/run_action.py \
```

Comment:
Hi @renefritze what if `GITHUB_ACTION_PATH` contains a space or a special character like `?` or `*`?

Reply:
Has this been addressed @renefritze btw?

Reply:
Quoted the paths, yes.